stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -405,15 +405,15 @@ sparse(a::AbstractVector) = sparsevec(a)
 
 function _dense2indval!(nzind::Vector{Ti}, nzval::Vector{Tv}, s::AbstractArray{Tv}) where {Tv,Ti}
     require_one_based_indexing(s)
-    cap = length(nzind);
+    cap = length(nzind)
     @assert cap == length(nzval)
     n = length(s)
     c = 0
     @inbounds for i = 1:n
         v = s[i]
```

Comment:
How about making this
```suggestion
    @inbounds for (i, v) in enumerate(s)
```
?

Reply:
Done